### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by danielnordh**
Update the outdated content list to reflect the actual state of the guide.
Add note on the announcement of the guide under Origin and History.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,86 +22,67 @@ The guide is meant to be a simple starting-point for anyone looking to build Bit
 
 The guide is still being created and should be seen as an evolving resource. Similar to the iterative design process, it starts with core parts then grows in ambition based on community support.
 
-Here is an initial outline that will be updated as needed, content that is live is linked:
+Here is an outline of the content that is currently live:
```
Comment (GBKS) [inline README.md:25]: Do we even need to repeat this table of contents here? Can we remove it?

Comment (pavlenex) [reply README.md:25]: ACK for removing it.

Comment (danielnordh) [reply README.md:25]: Good point. I guess we can remove it.
Alternative we just keep the top level list with links to the chapters?

Comment (GBKS) [reply README.md:25]: I have no strong opinion either way. Feel free to decide and then we can merge this update.

Comment (danielnordh) [reply README.md]: Removed everything but the chapter links.

**Review by pavlenex — APPROVED** [review]

**Pull request merged**